packages/react/webpack.config.js

```diff
@@ -21,6 +21,14 @@ module.exports = function config({ config: webpackConfig }) {
     });
   }
 
+  // Workaround for npm's fs package
```

Comment:
Have you verified that this works in your project? If so, can you add a comment here indicating that this is "so that webpack can handle *.json files."

https://github.com/request/request/issues/1529#issuecomment-103454943